js/compiler/igor-controller.js

```diff
@@ -72,20 +79,43 @@ function job_remove(job) {
  * Select the flags for Igor to run the job.
  * @param {GMLProject} project
  * @param {string} runtime_path
+ * @param {string?} user_path
  * @param {IgorSettings} settings
  * @returns {Result<string[]>}
  */
-function job_flags_get(project, runtime_path, settings) {
+function job_flags_get(project, runtime_path, user_path, settings) {
+
+    /**
+     * @type {{[K in IgorPlatform]?: string}}
+    */
+    const exts = {
+        Windows: ".zip",
+        Mac: ".zip",
+        Linux: ".appimage",
+    };
+
+    let projectName = $gmedit['gml.Project'].current.name;
+    // Remove .yyp file extension
+    projectName = projectName.substring(0, projectName.length - 4);
+
     const flags = [
         `/project=${project.path}`,
         `/config=${settings.configName}`,
         `/rp=${runtime_path}`,
-        `/runtime=${settings.runtime}`
+        `/runtime=${settings.runner}`,
+        `/v`,
+        `/tf=${project.dir}/output/${projectName}${exts[igor_platform_cmd_name] ?? ""}`
```

Comment:
GMEdit exposes a `displayName` and `name` if you query the project, see here:
https://github.com/thennothinghappened/GMEdit-Constructor/blob/main/js/global.d.ts#L369